File: Office.API/Models/Worker.cs
using System;

namespace Office.API.Models
{
    public class Worker
    {      
        public int Id { get; set; }
        public string Name { get; set; }
        public string Surname { get; set;}
        public DateTime LastLogin { get; set; }
        public string JobPosition {get; set;}
        public string Supervisor { get; set; }
        public bool CheckedIn {get;set;}
        public int UserID {get; set;}
        public bool unReadMessage {get; set;}
        public int OfficeID {get; set;}

    }
}

File: Office-SPA/src/app/_models/worker.ts

export interface Worker {
    id: number;
    name: string;
    surname: string;
    supervisor: string;
    checkedIn: boolean;
    userID: number;
    jobPosition: string;
    lastLogin: Date;
    OfficeID: number;
}

export interface WorkerCreate {
    name: string;
    surname: string;
    supervisor: string;
    checkdIn: boolean;
    password: string;
    email: string;
    username: string;
    OfficeID: number;
}
